packages/api/src/db/comment.ts

```diff
@@ -0,0 +1,93 @@
+import {RichTextNode} from '../graphql/richText'
+import {SortOrder, Limit, InputCursor, ConnectionResult} from './common'
+
+export enum CommentStatus {
```

Comment:
use camelCase for the value of the enums.

Reply:
Also please name it `CommentState`

Reply:
fixed